navidrome/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  navidrome:
+    image: deluan/navidrome:latest
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+    ports:
+      - "4533:4533"
```

Comment:
If this port is only being used to serve http(s) (which I think it does) it is better to use the app proxy (see line 2). In this case we can omit those ports here:
```suggestion
```